build/Targets/Imports.targets

```diff
@@ -507,4 +507,7 @@
     <!-- Special items in VSSDK projects -->
     <None Include="@(VSCTCompile)" />
   </ItemGroup>
+  
+  <!-- Work around https://github.com/dotnet/roslyn/issues/20945 -->
+  <Target Name="_CheckForUnsupportedNETCoreVersion" />
```

Comment:
Don't think so.

The problem here is that we're doing, and have been doing for some time, something that is not supported: targeting NetCore 2.0 from 1.1 SDK. Up until today there was nothing that said we shoudln't be doing that but the SDK team was surprised it ever worked at all. Had we gotten this warning immediately we could have avoided the need to add this hack now. 

Prefer to keep the other warning because it would stop us from taking a new unsupported depnedency.